mysql-test/include/have_rocksdb_as_default.inc

```diff
@@ -0,0 +1,12 @@
+--source include/have_rocksdb.inc
```

Comment:
Yes, everything you suggest sounds good. 

BTW, if you are patching the testcases for s/have_rocksdb/have_rocksdb_as_default/g, then patching for SET default_storage_engine=... is similar amount of patching, isn't it :) I'm not saying to make this change, it's just something that could be considered in the future